src/models.mysql/Participant.js

```diff
@@ -0,0 +1,56 @@
+const Sequelize = require('sequelize');
+
+module.exports = function(sequelize, DataTypes) {
+  return sequelize.define('Participant', {
```

Comment:
`Participant` can be a bit misleading name for this table. I first thought that this table represents people, but in fact it's table-helper to support M:N relation.